ci/CMakeLists.txt

```diff
@@ -37,14 +37,18 @@ add_subdirectory("../Jucer2Reprojucer/generated/JUCE/examples/audio plugin host"
   "Plugin Host"
 )
 
+if(NOT MSVC_VERSION EQUAL 1900)
```

Comment:
`NetworkGraphicsDemo.jucer` doesn't have a Visual Studio 2015 exporter, so we cannot build it with `Reprojucer.cmake` when using Visual Studio 2015 (i.e. `MSVC_VERSION == 1900`).